applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -275,6 +425,17 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 		return;
 	}
 
+	headsets[channel_index].pref_qos = *pref;
```

Comment:
This needs to be specific for the given stream, source/sink

Reply:
It is, each audio receiver will have a a preferred QoS (*pref is a pointer to the devices preferred QoS). It is linked to the stream in the headset[]. 
Not sure what you are asking here?

Reply:
You have one pref_qos for the headset though, there can be two streams. So you get here while configuring sink_stream and you store the pref_qos in headset[0].pref_qos, then source_stream is being configured and you overwrite headset[0].pref_qos with the *pref from that stream. These preferred QoS can be very different